src/main/java/mate/academy/springboot/datajpa/mapper/ProductMapper.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.springboot.datajpa.mapper;
+
+import mate.academy.springboot.datajpa.dto.request.ProductRequestDto;
+import mate.academy.springboot.datajpa.dto.response.ProductResponseDto;
+import mate.academy.springboot.datajpa.model.Product;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductMapper implements RequestDtoMapper<ProductRequestDto, Product>,
```

Comment:
```suggestion
public class ProductDtoMapper implements RequestDtoMapper<ProductRequestDto, Product>,
```
same with category, we can map product into different things, let's be more specific